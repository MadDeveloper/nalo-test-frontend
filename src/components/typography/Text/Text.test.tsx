import renderer from "react-test-renderer"
import { Text } from "."

describe("Components > Text", () => {
  it("should match snapshot", () => {
    const tree = renderer
      .create(
        <div>
          <Text>Lorem ipsum</Text>
          <Text secondary>Lorem ipsum</Text>
          <Text italic>Lorem ipsum</Text>
          <Text underlined>Lorem ipsum</Text>
          <Text uppercased>Lorem ipsum</Text>
          <Text medium>Lorem ipsum</Text>
          <Text semibold>Lorem ipsum</Text>
          <Text bold>Lorem ipsum</Text>
          <Text heavy>Lorem ipsum</Text>
          <Text black>Lorem ipsum</Text>
        </div>
      )
      .toJSON()

    expect(tree).toMatchSnapshot()
  })
})
