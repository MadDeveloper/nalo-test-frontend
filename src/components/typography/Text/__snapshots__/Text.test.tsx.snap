// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components > Text should match snapshot 1`] = `
<div>
  <span
    className="root"
  >
    Lorem ipsum
  </span>
  <span
    className="root secondary"
  >
    Lorem ipsum
  </span>
  <span
    className="root italic"
  >
    Lorem ipsum
  </span>
  <span
    className="root underlined"
  >
    Lorem ipsum
  </span>
  <span
    className="root uppercased"
  >
    Lorem ipsum
  </span>
  <span
    className="root medium"
  >
    Lorem ipsum
  </span>
  <span
    className="root semibold"
  >
    Lorem ipsum
  </span>
  <span
    className="root bold"
  >
    Lorem ipsum
  </span>
  <span
    className="root heavy"
  >
    Lorem ipsum
  </span>
  <span
    className="root black"
  >
    Lorem ipsum
  </span>
</div>
`;
